deploy/examples/operator-openshift.yaml

```diff
@@ -639,6 +640,9 @@ spec:
             runAsNonRoot: true
             runAsUser: 2016
             runAsGroup: 2016
+            capabilities:
+              drop:
+                - ALL
```

Comment:
We don't need this on the operator pod since it's on the scc, right?

Reply:
This should instead be added in operator.yaml, right?

Reply:
yeah, that's right modified that

Reply:
Have you verified this?